src/api/userAPI.ts

```diff
@@ -0,0 +1,29 @@
+import { REACT_APP_API_URL } from '@env';
+import axios from 'axios';
+import jwt_decode from 'jwt-decode';
+
+import { UserType } from '../types/types';
+
+interface ILogin {
+   token: string
+}
+
+export const registration = async (email, password): Promise<UserType | void> => {
+  try {
+    const { data } = await axios.post<ILogin>(`${REACT_APP_API_URL}api/user/registration`, { email, password });
```

Comment:
Create a separate file called networkClient.js
Inside of this file you import axios.
Then you specify BASE_URL for the axios.
Then you export axios and import it from networkClient.js, not the 'axios' library.